### PR TITLE
Add ropemaking proficiencies and smooth out ropemaking recipes

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
SUMMARY: Content "Adds ropemaking proficiencies and more convenient ropemaking recipes"

#### Purpose of change
The basic proficiency framework was completed in #42476 but we still don't have many of these. At some point most complex actions should be linked to a proficiency, and we need to start expanding the proficiency framework to see the gaps remaining.

#### Describe the solution

1. Adds a basic 'fibers' proficiency representing a general functional knowledge of how fibers are twisted. This is easy to learn.  
2. Adds a more complex 'ropemaking' proficiency, harder to learn, representing actual skill with the craft.
3. Adds some new rope recipes to reduce keypresses when making rope out of precursors.

#### Testing
Not yet

### CODE REVIEW

**Comment by FuelType-Memes** [comment]
Those time multipliers are getting brutal

**Comment by I-am-Erk** [comment]
It doesn't take long to learn basic fiber proficiency, but if you're just grabbing some cordage in the forest and trying to start from nothing, the first few tries will take you a lot of work. Basically, winding together a couple strings gets you that first prof.

**Comment by FuelType-Memes** [comment]
> 
> 
> It doesn't take long to learn basic fiber proficiency, but if you're just grabbing some cordage in the forest and trying to start from nothing, the first few tries will take you a lot of work. Basically, winding together a couple strings gets you that first prof.

Yes, I understand that. Hope you'll add basic cordmaking to survivalist professions

**Comment by Amoebka** [comment]
Can mutations teach proficiencies yet? It would be nice if arachnid mutants got these for free after they go post-threshold and develop "spider brains".

It wouldn't have any gameplay impact since they can just produce long ropes naturally, but it would be very flavourful.

**Comment by I-am-Erk** [comment]
Just finished updating professions in fact.

I don't think spider silk bears any real resemblance to rope spinning, otherwise that would be a fun idea.

**Comment by I-am-Erk** [comment]
I'm not able to test this on my own until... well, it might be a little while. If someone else is able to test and review it that would be neat but otherwise there's some chance I'll be able to from my hotel tonight.

File: data/json/professions.json [hunk 1]
```diff
@@ -3130,6 +3137,7 @@
     "name": "Naturalist",
     "description": "Over long years of self-imposed exile in the wilderness, you have come to an understanding with Mother Nature.  The world as they knew it might have ended for your forsaken species, but you can hardly tell the difference.",
     "points": 6,
+    "proficiencies": [ "prof_fibers", "prof_fibers_rope" ],
```
Comment (LaVeyanFiend) [inline data/json/professions.json:3140]: There's a second proficiencies line after the skills on line 3153, need to delete that one to load in.
```suggestion
    "proficiencies": [ "prof_fibers", "prof_fibers_rope", "prof_knapping" ],
```

**Review by LaVeyanFiend — COMMENTED** [review]
Tested as requested:
1) Attempted to load. Found the loading error on Naturalist having two ``"proficiencies"`` lines I left a code suggestion for. Game loads after applying fix.
2) Spawned in as a normal survivor, crafted short ropes with time penalty from not having the skill. Learned Fiber Twisting after crafting the short ropes before any XP in Ropemaking was acquired as expected. Restarted to remove Fiber Twisting, cannot skip to learning Ropemaking by using short->long rope recipe.
3) Spawned in as Naturalist(both proficiencies show in chargen screen), time penalties are removed as expected.

**Comment by I-am-Erk** [comment]
Thanks for that. Test results implemented.

**Pull request merged**